docs/basics/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting first time setup
+
+## My site is showing "Internal Server Error"
+
+If you are seeing a blank screen with just the words "Internal Server Error" on it,
+it means that the django webworker process is not reporting as "down",
+but still failing to bootstrap fully.
+(If you are seeing a more elaborate 500 page, then that is an issue with a single request,
+but ususally does not indicate a more pervasive problem with the site's ability to receive and handle requests in general.)
+Often this is because it is unable
+using a backing service (such as PostgreSQL, CouchDB, or Redis).
+This in term can fall into a number of categories of issue:
+1. the service is down
+2. the service is up, but unreachable due to network configuration
+3. the service is up and reachable, but is blocking the connection for a permissions-related reason
+   (auth is wrong, IP is not whitelisted, etc.)
+4. the service is up, reachable, and accepting the connection,
+   but is failing due to other problems such as misconfiguration (e.g. a low max connection limit),
+   or other problem such as out of memory errors.
+
+### Further diagnosis
+You can start by backing up and simply checking which of the application services are reporting as up
+by running
+
+```bash
+commcare-cloud <env> service commcare status
+```
+
+You will likely find the `django` process is reporting as RUNNING.
+Some other processes if affected by a similar issue may (or may not) be reporting as FATAL "exited too quickly".
+
+To dig into a particular error, you can log into the machine and tail one of the logs:
+
+```bash
+commcare-cloud <env> ssh webworkers:0
+$ tail -n100 /home/cchq/www/<env>/log/django.log
```

Comment:
I discovered that you can also do this with supervisorctl which doesn't require you finding the right log file:
```
sudo supervisorctl tail -f <process name>
```

Reply:
Whoa, that's a great tip right there!